documentation/modules/exploit/multi/php/car_rental_system.md

```diff
@@ -0,0 +1,76 @@
+## Vulnerable Application
+
+Car Rental System 1.0 is a web application that allows users to rent cars using an online portal. It was written using PHP and MySQL.
+
+Vendor Homepage: https://www.sourcecodester.com/php/14544/car-rental-management-system-using-phpmysqli-source-code.html
+Software Link: https://www.sourcecodester.com/download-code?nid=14544&title=Car+Rental+Management+System+using+PHP%2FMySQLi+with+Source+Code
+
+to install, extract all files in a web directory, edit admin/db_connect.php, and create database following database/car_rental_db.sql
+
+Users can log into the app as an administrator by using the username ` ' OR 1=1 limit 1#` , 
+and a random word as password. `limit 1` is used as it ensures the database will select the first
+user in the database, ADMIN, thereby logging the attacker in as the administrative user.
+
+Once authenticated as an administrator, attackers can then upload arbitrary files via the `/admin/ajax.php?action=save_car` page, which will then be placed under the `/admin/` directory on the web server. By uploading
+an executing a malicious PHP file, attackers can gain arbitrary code execution on the web server as the `www-data` user.
+
+## Verification Steps
+
+Confirm that functionality works:
+1. Start `msfconsole`
+2. `use exploit/multi/php/car_rental_system`
+3. Set the `RHOST`.
+4. Set the `TARGETURI`.
+7. Set `LHOST` and `LPORT`
+8. Run the exploit: `run`
+9. Confirm that you now have a shell on the target machine as the `www-data` user.
+
+**TARGETURI**
+The path to the Car Rental System HOME directory.
+
```

Comment:
Options should use the markdown header not bold. The extra blank line can be removed as well.

```suggestion
### TARGETURI
The path to the Car Rental System HOME directory.
```